doc/releases/changelog-0.30.0.md

```diff
@@ -149,53 +288,115 @@
+  be the _remaining_ time evolution complementary to the output for `complementary=False`.
+  See the [docstring](https://docs.pennylane.ai/en/stable/code/api/pennylane.pulse.ParametrizedEvolution.html)
+  for details.
+
+* Hardware-compatible pulse sequence gradients with `stoch_pulse_grad` can now be calculated faster using
+  the new keyword argument `use_broadcasting`. Executing a `ParametrizedEvolution` that returns
+  intermediate evolutions has increased performance using the state vector ODE solver, as well.
+  [(#4000)](https://github.com/PennyLaneAI/pennylane/pull/4000)
+  [(#4004)](https://github.com/PennyLaneAI/pennylane/pull/4004)
 
 <h4>Intuitive QNode returns</h4>
 
-* The default Gaussian device and parameter shift CV support the new return system, but only for single measurements.
-  [(3946)](https://github.com/PennyLaneAI/pennylane/pull/3946)
+* The QNode keyword argument `mode` has been replaced by the boolean `grad_on_execution`.
+  [(#3969)](https://github.com/PennyLaneAI/pennylane/pull/3969)
+
+* The `"default.gaussian"` device and parameter-shift CV both support the new return system, but only for single measurements.
+  [(#3946)](https://github.com/PennyLaneAI/pennylane/pull/3946)
 
 * Keras and Torch NN modules are now compatible with the new return type system.
   [(#3913)](https://github.com/PennyLaneAI/pennylane/pull/3913)
   [(#3914)](https://github.com/PennyLaneAI/pennylane/pull/3914)
 
-* `DefaultQutrit` supports the new return system.
+* `DefaultQutrit` now supports the new return system.
   [(#3934)](https://github.com/PennyLaneAI/pennylane/pull/3934)
 
-  [(3946)](https://github.com/PennyLaneAI/pennylane/pull/3946)
-
 <h4>Other improvements</h4>
 
-* Added a `Shots` class to the `measurements` module to hold shot-related data.
+* A `Shots` class has been added to the `measurements` module to hold shot-related data.
   [(#3682)](https://github.com/PennyLaneAI/pennylane/pull/3682)
 
-* The `coefficients` function and the `visualize` submodule of the `qml.fourier` module
-  now allow assigning different degrees for different parameters of the input function.
-  [(#3005)](https://github.com/PennyLaneAI/pennylane/pull/3005)
-
-  The arguments `degree` and `filter_threshold` to `qml.fourier.coefficients` previously were
-  expected to be integers, and now can be a sequences of integers with one integer per function
-  parameter (i.e. `len(degree)==n_inputs`), resulting in a returned array with shape
-  `(2*degrees[0]+1,..., 2*degrees[-1]+1)`.
-  The functions in `qml.fourier.visualize` accordingly accept such arrays of coefficients.
-
-* `Operator` now has a `has_generator` attribute that returns whether or not the operator
-  has a generator defined. It is used in `qml.operation.has_gen`, improving its performance.
-  [(#3875)](https://github.com/PennyLaneAI/pennylane/pull/3875)
-
 * The custom JVP rules in PennyLane now also support non-scalar and mixed-shape tape parameters as
```

Comment:
Not sure I get this one 🤔

Reply:
> **Benefits:**
> More flexible JVPs. In particular, the option to differentiate with the interfaces while using non-scalar differentiable > tape parameters, and when using multi-dimensional tape returns like e.g. broadcasted probs

Does this help? Maybe @dwierichs can say more.